exec.go

```diff
@@ -115,7 +116,7 @@ func execProcess(context *cli.Context) (int, error) {
 		return -1, err
 	}
 	r := &runner{
-		enableSubreaper: !context.Bool("no-subreaper"),
+		enableSubreaper: false,
```

Comment:
ya, just thought i would make it explicit. it was not omitted but set that way for a reason